RetailCoder.VBE/API/ParserState.cs

```diff
@@ -28,30 +77,28 @@ public interface IParserState
     [ComVisible(true)]
     [Guid("3D8EAA28-8983-44D5-83AF-2EEC4C363079")]
     [InterfaceType(ComInterfaceType.InterfaceIsIDispatch)]
-    public interface IParserStateEvents
+    public interface IRubberduckParserStateEvents
     {
-        void OnParsed();
-        void OnReady();
-        void OnError();
+        void OnStateChanged(ParserState state);
```

Comment:
The events don't work because of some threading issue. I wanted to expose different handlers so the VBA client code could implement different code in a different handler, making a simpler API (no need to switch on the state) - why replace them all with one?